### DISCUSSION TITLE
Connectivity with Debrid cloud, and linking.

### DISCUSSION THREAD

**Discussion by sirensongss** [Goldenfreddy0703/Otaku#96] [Ideas]
A feature like Fen's Link to Movie/TV show would be really appreciated. I found a cut of a show, and I'd like to have it tracked and scrobbled. The ability to view my RD cloud from Otaku, and link it to shows would be a huge help.

**Comment by Goldenfreddy0703** [comment]
Sounds like a good feature but we would have to add that to Real-Debrid. Premiumize, All-Debrid, and maybe Easynews if we are thinking of adding it.

**Comment by Goldenfreddy0703** [comment]
make this a feature request and will see if we can look into it someday. Tracking may not be as good tho unfortunately if we were to add this.
